contracts/src/v0.8/keystone/CapabilityRegistry.sol

```diff
@@ -251,6 +259,9 @@ contract CapabilityRegistry is OwnerIsCreator, TypeAndVersionInterface {
   /// Deprecated capabilities are skipped by the `getCapabilities` function.
   EnumerableSet.Bytes32Set private s_deprecatedHashedCapabilityIds;
 
+  /// @notice Encoded node signer addresses
+  EnumerableSet.Bytes32Set private s_encodedNodeSignerAddresses;
```

Comment:
```suggestion
  EnumerableSet.Bytes32Set private s_nodeSigners;
```

The fact that it is an encoded address leaks EVM details. It also doesn't mirror Node.signer